### DISCUSSION TITLE
Google log in pop up blocks a click

### DISCUSSION THREAD

**Discussion by facundozupel** [daijro/camoufox#146] [Q&A]
Any ideas on how to avoid Google´s pop up for log in? also, any ideas on how to click on the exit button?

**Comment by KrAsH-CoD3** [comment]
> Any ideas on how to avoid Google´s pop up for log in? also, any ideas on how to click on the exit button?

Can you be more specific at what page?

**Reply by facundozupel** [reply]
I´m trying to log in on reddit. The pop up blocks the click i´m trying to execute.
<img width="698" alt="Captura de pantalla 2024-12-21 a la(s) 3 57 08 p  m" src="https://github.com/user-attachments/assets/57436a4c-cce5-4660-9ad6-ee488727238f" />
